.github/workflows/ci.yml

```diff
@@ -18,9 +18,15 @@ jobs:
         with:
           username: fedimintui
           password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
-      - name: Build and push
+      - name: Build and push guardian-ui
         uses: docker/build-push-action@v4
         with:
           file: Dockerfile.guardian-ui
           push: true
           tags: fedimintui/guardian-ui:latest
+      - name: Build and push gateway-ui
+        uses: docker/build-push-action@v4
+        with:
+          file: Dockerfile.gateway-ui
+          push: true
+          tags: fedimintui/gateway-ui:latest
```

Comment:
Does it make sense to have 2 different workflows for these images, and to have them configure `paths` to reduce unnecessary runs or identical docker images? Something like:
```yml
# .github/workflows/guardian-ui.yml
on:
  push:
    branches:
      - master
    paths:
      - .github/workflows/guardian-ui.yml
      - ui/apps/guardian-ui/**
      - ui/packages/**
```
and
```yml
# .github/workflows/gateway-ui.yml
on:
  push:
    branches:
      - master
    paths:
      - .github/workflows/gateway-ui.yml
      - ui/apps/gateway-ui/**
      - ui/packages/**
```